src/js/index.js

```diff
@@ -1,17 +1,129 @@
-const $showResultButton = document.querySelector('.open-result-modal-button')
-const $modalClose = document.querySelector('.modal-close')
-const $modal = document.querySelector('.modal')
-const $lottoNumbersToggleButton = document.querySelector(
-  '.lotto-numbers-toggle-button'
-)
+import { isValidForNoAmount, isValidForExactAmount, isAlreadyExist } from './validators.js';
+import LottoModel from './lotto.js';
+import { generateLottoNumbers } from './generateLottos.js';
+import { MESSAGE_FOR_EMPTY_VALUE, MESSAGE_FOR_INVALID_UNIT_VALUE } from './constants.js';
+import { closeModal, displayDetails, openModal } from './views.js';
+import {
+  addLottoToggleButtonClickEventListener,
+  addPurchaseButtonClickEventListener,
+  addResultButtonClickEventListener,
+  addModalCloseClickEventListener,
+} from './eventListeners.js';
+
+const $purchaseAmountInput = document.querySelector('#purchaseAmount');
+const $totalQuantity = document.querySelector('#totalQuantity');
+const $lottoIconList = document.querySelector('#lottoIconList');
+
+const lottoState = {
+  purchasedAmount: 0,
+  quantity: 0,
+  lottos: [],
+  isOpen: false,
+};
+
+const onToggleChange = () => {
```

Comment:
```suggestion
const handleToggleButton = () => {
```
보통 이렇게 표현한답니다!

handle + 행위 + 대상

Reply:
무심코 회사 컨벤션과 비슷하게 따라가버렸네요...
원래는 handle이 저도 익숙했었는데ㅠ
suggestion 감사합니다!

Reply:
[c406d2b](https://github.com/next-step/js-lotto/pull/166/commits/c406d2b8f7a8f64b9eedaceae6f2e7ba2334dd8d)로 반영해보았습니다!